packages/editor-ui/src/components/RunDataTable.vue

```diff
@@ -408,24 +406,6 @@ export default mixins(externalHooks).extend({
 				entryRows = [];
 				leftEntryColumns = Object.keys(entry || {});
 
-				// Go over all the already existing column-keys
-				tableColumns.forEach((key) => {
-					if (entry.hasOwnProperty(key)) {
-						// Entry does have key so add its value
-						entryRows.push(entry[key]);
-						// Remove key so that we know that it got added
-						leftEntryColumns.splice(leftEntryColumns.indexOf(key), 1);
-
-						hasJson[key] =
-							hasJson[key] ||
```

Comment:
why are you removing this logic? this is still used to determine max width of columns

Reply:
As far as I understood it never runs, just a couple of lines before is `tableColumns` defined, and at this point, it is always an empty array so calling foreach on it doesn't have any effect

Reply:
![image](https://user-images.githubusercontent.com/5410822/195830027-05d09933-55e4-4500-ad7c-16cf9471d470.png)

Reply:
no ofc it runs... but not in the first run

Reply:
I see, my bad, the whole block is inside a `forEach`, I'll add it back